tests/test_hparams.py

```diff
@@ -34,6 +34,7 @@ def _configure_dataset_for_synthetic(dataset_hparams: DatasetHparams) -> None:
 
 @pytest.mark.parametrize("hparams_file", walk_model_yamls())
 class TestHparamsCreate:
+    pytest.importorskip("timm")  # yapf: disable
```

Comment:
Based on the docs for `pytest.importorskip`, i think this test will be now be skipped for every model, not just the ones that require `timm`.

Reply:
you're right! I missed that. fixing.